.gitignore

```diff
@@ -79,6 +79,7 @@ Session.vim
 out/
 .idea_modules/
 atlassian-ide-plugin.xml
+mocha.iml
```

Comment:
what is this?

Reply:
that's a JetBrains-related file. but when I used it, it wanted to put them in `.idea`.  

@sstephant I think this is a non-standard location for the `.iml` file?  If so, maybe better to add it to your _global_ gitignore file.

Reply:
Yes it's Jetbrains related. I'm using IntelliJ IDEA which generates this in addition to the .idea folder (unlike PhpStorm for example). I can remove it from the .gitignore.